### ISSUE TITLE
Planning

### ISSUE THREAD

**Issue by hscannell** [hscannell/marineHeatwave-image-classification#1]
The objective of this hackathon is to predict the evolution of marine heatwaves in the North Pacific (20ºN–70ºN) using images of meteorological and ocean surface variables. 

**Inputs:**

- [x] Sea Surface Temperature ([OISSTv2.1](https://www.ncdc.noaa.gov/oisst/optimum-interpolation-sea-surface-temperature-oisst-v21))
- [x] Labeled images of marine heatwaves from [Ocetrac](https://ocetrac.readthedocs.io/en/latest/) using OISSTv2.1
- [ ] Meteorological variables ([OAFlux](https://oaflux.whoi.edu/)) 

Monthly inputs will likely span from 1988–2020 and be on a 0.25º resolution grid. We can use [xESMF](https://xesmf.readthedocs.io/en/latest/) to regrid if necessary. 

**Output:**

- Classification at each grid point at a future time of whether or not a marine heatwave is present.

**Evaluation Metric:**

- F1: Precision and Recall 

**Anticipated Challenges:**
- Class imbalance

**Comment by hscannell** [comment]
I also suppose that we **do not** want to detrend any of the data. Can you confirm @geo-yrao?

**Comment by hscannell** [comment]
The netCDF of Ocetrac labels is approximately 3.7 GB.

**Comment by geo-rao** [comment]
> I also suppose that we **do not** want to detrend any of the data. Can you confirm @geo-yrao?

Yes, we don't want to detrend any of the data. But we can add a short section of how to do that in the entry notebook as a possible exploration.

**Comment by geo-rao** [comment]
> The netCDF of Ocetrac labels is approximately 3.7 GB.

I think this should be fine. It might be better if we upload on zenodo and share the direct doi link for the data.

**Comment by wcneill** [comment]
So I found this repo super by chance. Any chance I can contribute? I just signed up for the workshop. I'm a mathemetician and a very junior data scientist in an unrelated field, but I want to get into applying ML to environmental science. I don't have much domain knowledge, but I'm hoping to pick it up along the way.

**Comment by geo-rao** [comment]
> So I found this repo super by chance. Any chance I can contribute? I just signed up for the workshop. I'm a mathemetician and a very junior data scientist in an unrelated field, but I want to get into applying ML to environmental science. I don't have much domain knowledge, but I'm hoping to pick it up along the way.

Hi @wcneill, sorry for just seeing this. Yes, definitely! You are welcomed to join the hackathon. We will be sending out an email with instructions this Wednesday with more instructions to join the hackathon. Please stay tuned! Also, feel free to reach out to me via [email](douglas.rao@noaa.gov).
